testing/ncids-css-testing/stories/uswds-native/usa-collection/src/index.scss

```diff
@@ -0,0 +1,6 @@
+@use "styles/ncids";
+@forward "uswds-fonts";
```

Comment:
uswds-fonts is never needed as a separate import. it is imported by uswds-global.